Product_Landing_Page.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="Product_Landing_Page.css">
+    <title>Document</title>
+</head>
+<body>
+       <div>
+        <header id="header">
+            <img id="header-img"src="https://cdn.freecodecamp.org/testable-projects-fcc/images/product-landing-page-logo.png" alt="Company Logo here">
+            <nav id="nav-bar">
+                <ul>
+                    <li><a class="nav-link"href="index.html#section1">ABOUT</a></li>
+                    <li><a class="nav-link"href="#section2">APARTMENTS</a></li>
+                    <li><a class="nav-link"href="index.html#section3">LANDLORDS</a></li>
+                    <li><a class="nav-link"href="#section4">CONTACT</a></li>
+                </ul>
+            </nav>
+            <nav>
+                <ul>
+                    <li><a href="#section1">ABOUT</a></li>
+                    <li><a href="#section2">APARTMENTS</a></li>
+                    <li><a href="#section3">LANDLORDS</a></li>
+                    <li><a href="#section4">CONTACT</a></li>
+                </ul>
+            </nav>
+            <form id="form" action="https://www.freecodecamp.com/email-submit" method="post">
+                <input id="email" type="email" name="email" href="#"placeholder="email">
+                <input type="submit" id="submit" value="submit" type="email" name="email" required />
+            </form>
+        </header>
+    </div>
+    <section class="section1">
+        <h1>This is a header</h1>
+    </section>
+   <section class="section2">
+    <iframe id="video"width="560" height="315" src="https://www.youtube.com/embed/egS3ykrYvio" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+   </section>
+   <section>
+       <img src="https://media.istockphoto.com/photos/aerial-view-of-mountain-bikers-in-small-trails-in-the-mountain-picture-id1310492435" alt="">
+   </section>
+    <footer>
+        
+    </footer>
```

Comment:
I believe that since there is nothing in the footer element, it should be removed.